components/filter-box.js

```diff
@@ -1,5 +1,4 @@
 import React from 'react'
-import "../styles/index.css";
```

Comment:
This was the problem line. This isn't how you import styles in tailwinds from a component. [See this guide](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css). Components get their own styles, but you're importing a "global" style here.